bahmni-docker/README.md

```diff
@@ -272,12 +285,42 @@ By default, the configuration of openmrs and openmrsdb services are set to load
  | REPORTS_DB_USERNAME | Username of Reports Database                                                                                                                                                        |
  | REPORTS_DB_PASSWORD | Password of Reports Database                                                                                                                                                        |
 
-## Appointments Configuration
+## Appointments Configurations
 | Variable Name          | Description                                                                                                                                                                                                    |
 |:-----------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | APPOINTMENTS_IMAGE_TAG | This value specifies which image version needs to be used for appointments service. List of tags can be found at [bahmni/appointments - Tags](https://hub.docker.com/r/bahmni/appointments/tags) .             |
 | APPOINTMENTS_PATH      | Set this to the directory path where you have cloned  [openmrs-module-appointments-frontend](https://github.com/Bahmni/openmrs-module-appointments-frontend) repository. Needed only for development purposes. |
 
+## PACS DB Configurations
+| Variable Name         | Description                                                                                |
+|:----------------------|:-------------------------------------------------------------------------------------------|
+| PACS_DB_HOST          | The host name of the postgres server which runs database for dcm4chee and pacs-integration |
+| PACS_DB_PORT          | The port of the postgres server for dcm4chee and pacs-integration                          |
+| PACS_DB_ROOT_PASSWORD | The root password given to the postgres container ie pacsdb service                        |
+
+## DCM4CHEE Configurations
+| Variable Name        | Description                                                                                                                                                                           |
+|:---------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| DCM4CHEE_IMAGE_TAG   | This value specifies which image version needs to be used for reports service. List of tags can be found at [bahmni/dcm4chee - Tags](https://hub.docker.com/r/bahmni/dcm4chee/tags) . |
```

Comment:
"reports service" (seems typo). Should be dcm4chee service.

Reply:
Ahh. Yes Copy-paste issue :). Will fix